controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -385,59 +388,51 @@ def test_create_volume_succeeds(self, storage_agent):
         response_volume = self.servicer.CreateVolume(self.request, self.context)
         self.assertEqual(self.context.code, grpc.StatusCode.OK)
         self.mediator.get_volume.assert_called_once_with(volume_name, pool_id='pool1')
-        self.mediator.create_volume.assert_called_once_with(volume_name, 10, {}, 'pool1')
+        self.mediator.create_volume.assert_called_once_with(volume_name, 10, None, 'pool1')
         self.assertEqual(response_volume.volume.content_source.volume.volume_id, '')
         self.assertEqual(response_volume.volume.content_source.snapshot.snapshot_id, '')
 
     @patch("controller.controller_server.csi_controller_server.get_agent")
     def test_create_volume_with_wrong_secrets(self, a_enter):
-        self._test_create_object_with_wrong_secrets(a_enter)
+        self._test_request_with_wrong_secrets(a_enter)
 
     @patch("controller.controller_server.csi_controller_server.get_agent")
     def test_create_volume_with_wrong_parameters(self, storage_agent):
-        storage_agent.return_value = self.storage_agent
-
-        self.request.parameters = {"pool": "pool1"}
-        self.servicer.CreateVolume(self.request, self.context)
-        self.assertNotEqual(self.context.code, grpc.StatusCode.INVALID_ARGUMENT)
-
-        self.request.parameters = {"capabilities": ""}
-        self.servicer.CreateVolume(self.request, self.context)
-        self.assertEqual(self.context.code, grpc.StatusCode.INVALID_ARGUMENT, "capacity is missing in secrets")
-        self.assertEqual(self.context.code, grpc.StatusCode.INVALID_ARGUMENT, "pool parameter is missing")
-        self.assertTrue("parameter" in self.context.details)
+        self._test_request_with_wrong_parameters(storage_agent)
 
     @patch("controller.controller_server.csi_controller_server.get_agent")
     def test_create_volume_with_wrong_volume_capabilities(self, storage_agent):
         storage_agent.return_value = self.storage_agent
 
-        caps = Mock()
-        caps.mount = Mock()
-        caps.mount.fs_type = "ext42"
-        access_types = csi_pb2.VolumeCapability.AccessMode
-        caps.access_mode.mode = access_types.SINGLE_NODE_WRITER
-
+        caps = utils.get_mock_volume_capability_object(fs_type="ext42")
         self.request.volume_capabilities = [caps]
 
         self.servicer.CreateVolume(self.request, self.context)
-        self.assertEqual(self.context.code, grpc.StatusCode.INVALID_ARGUMENT, "wrong fs_type")
+        self.assertEqual(self.context.code, grpc.StatusCode.INVALID_ARGUMENT)
         self.assertTrue("fs_type" in self.context.details)
 
-        caps.mount.fs_type = "ext4"
-        caps.access_mode.mode = access_types.MULTI_NODE_SINGLE_WRITER
+        access_types = csi_pb2.VolumeCapability.AccessMode

```

Comment:
I know it existed before your change, but please change `access_types` to `access_mode`

Reply:
done